libraries/WiFi/src/WiFiSTA.h

```diff
@@ -114,11 +116,14 @@ class WiFiSTAClass {
     const char *ca_pem = NULL, const char *client_crt = NULL, const char *client_key = NULL, int ttls_phase2_type = -1, int32_t channel = 0,
     const uint8_t *bssid = 0, bool connect = true
   );
+#if CONFIG_ESP_WIFI_ENTERPRISE_SUPPORT
```

Comment:
this should be 4 lines above where that previous begin is and also add the guards to it's implementation in `WiFioSTA.cpp`

Reply:
Thx a lot for helping. Looks way more correct now, even for someone like me not knowing what doing ;-)